apps/server/src/modules/user/service/user-query.type.ts

```diff
@@ -3,4 +3,5 @@ import { UserDO } from '@shared/domain/domainobject/user.do';
 export type UserQuery = Partial<Pick<UserDO, 'schoolId'>> & {
 	isOutdated?: boolean;
 	lastLoginSystemChangeGreaterThan?: Date;
+	outdatedSinceEquals?: Date;
```

Comment:
outdatedSince = Date ok but what is mean with Equals? If it should exact match the date when the since the user is outdated i think a better wording should be find. But this sound like a strong uc.

Reply:
Absolutelya correct. Removed it and usingf no the userDO field